applications/hips/README.md

```diff
@@ -15,7 +15,8 @@ HiPS tile server backed by Google Cloud Storage
 | autoscaling.maxReplicas | int | `100` | Maximum number of hips deployment pods |
 | autoscaling.minReplicas | int | `1` | Minimum number of hips deployment pods |
 | autoscaling.targetCPUUtilizationPercentage | int | `80` | Target CPU utilization of hips deployment pods |
-| config.gcsBucket | string | None, must be set | Name of Google Cloud Storage bucket holding the HiPS files |
+| config.datasets | string | None, must be set | A mapping of dataset names to locations in GCS. /api/hips/v2/<dataset name> will serve files out of the corresponding GCS bucket. |
```

Comment:
I'd make that `dataset-name` just because spaces outside quotations scare me, but I realize this is just a documentation string.